### PR TITLE
Ubuntu base

### PR DESCRIPTION
**Pull request by cdbishop**
@mattyclarkson ok If I merge this?
I've tested this locally running a tool-chain and it configures and builds fine.

### CODE REVIEW

**Comment by mattyclarkson** [comment]
@cdbishop, do we need to do your update/upgrade after the install of the `base-files`? I think I actually tried that and it didn't seem to update anything. Happy for merge though if it works in the TC! yay, saveing about 1.4GB!

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM vcatechnology/base-linux-mint
+FROM vcatechnology/ubuntu
```
Comment (mattyclarkson) [inline Dockerfile]: Ooooo, need to put `:16.04` here to make sure we don't break!

Comment (cdbishop) [reply Dockerfile]: Oh yea. Done.

**Comment by mattyclarkson** [comment]
OK, merged. Have updated the `18` branch to point to `master`. Need to implement the `17.3` branch. Which should be pretty simple.

**Pull request merged**